hnn_core/cell_response.py

```diff
@@ -381,7 +384,34 @@ def plot_spikes_hist(self, trial_idx=None, ax=None, spike_types=None,
         return plot_spikes_hist(self, trial_idx=trial_idx, ax=ax,
                                 spike_types=spike_types, show=show)
 
-    def write(self, fname):
+    def _write_hdf5(self, fname):
+        """Write spiking activity to a hdf5 file.
+
+        Parameters
+        ----------
+        fname : str
+            String format (e.g., 'spk.hdf5') of the
+            path to the output spike file.
+
+        Outputs
+        -------
+        A hdf5 file containing the cell response object
+        """
+        fname = str(fname)
+        # Hardcoded fname for test purposes
+        # Write code to check available file names
+        # upon discussion of the file name and location
+        print(f'Writing file {fname}')
+        data = {}
```

Comment:
```suggestion
        data = dict()
```

Nitpick but usually it's more clear to say `dict()` and `list` since `{}` and `[]` look very similar and can be confused.